### ISSUE TITLE
New WCAG Criteria -  2.4.13 Focus Appearance - AAA

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2941]
We know we have some missing visible indicators, but we also need to check some for this criteria: is at least as large as the area of a 2 CSS pixel thick perimeter of the unfocused component or sub-component, and has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.

**Comment by greensomes** [comment]
@lpgoldstein, Can you confirm if this issue is resolved with the adoption of the USWDS focus indicator.  

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOTBSQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7ff0f815fe1edc9820878f045b3ded85af00bb1e/image.png" alt="image.png" />

**Comment by lpgoldstein** [comment]
![image](https://github.com/user-attachments/assets/a9500d58-0a61-47ee-ad2a-f17051e7ce8e)
@greensomes The line is thick enough and I think the contrast is okay because the light blue is on a white background.The contrast between the blues is only 2:1 which is not enough, but the contrast between the white and blue outline is just over 3:1
